apis/beacon/blocks/attestations.yaml

```diff
@@ -25,9 +28,13 @@ get:
               finalized:
                 $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Finalized"
               data:
-                type: array
-                items:
-                  $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Attestation'
+                oneOf:
```

Comment:
here it does not make a difference if we use `anyOf` now instead which I think we should prefer to avoid [previous issue](https://validator.swagger.io/validator/debug?url=https%3A%2F%2Fethereum.github.io%2Fbeacon-APIs%2Freleases%2Fv2.4.2%2Fbeacon-node-oapi.json)

Reply:
done